src/scripts/mirror_signature_via_oak.py

```diff
@@ -0,0 +1,67 @@
+"""An alternative way of creating a `reports/mirror_signature-ONTOLOGY.tsv` which uses OAK instead of `robot`.
```

Comment:
FYI: The script which creates a `mirror_signature-%.tsv` via OAK instead of SPARQL/`robot`.